d3a_interface/schemas.py

```diff
@@ -79,7 +79,7 @@ class ScenarioSchemas:
                     "number_of_clones": {"type": "number"},
                     "uuid": {"type": "string"},
                     "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
-                    "initial_soc": {"type": "number"},
+                    "initial_soc": {"anyOf": [{"type": "string"}, {"type": "null"}]},
```

Comment:
I am a bit surprised that the soc is not sent as a float but as a string. As an extra precaution and in order to protect ourselves from changes in the future, I would recommend to also allow the number type.

Reply:
I actually replaces the string with number, Dont know why this was there.
Also tried it locally, works.